index.js

```diff
@@ -739,9 +704,16 @@ RedisClient.prototype.send_command = function (command, args, callback) {
 
     // if the value is undefined or null and command is set or setx, need not to send message to redis
     if (command === 'set' || command === 'setex') {
-        if(args[args.length - 1] === undefined || args[args.length - 1] === null) {
+        if (args.length === 0) {
```

Comment:
Since there are still some open issues about `undefined` and `null` values passed into redis we're going to have to have a closer look at this later on anyway. Therefor I'll merge it as is.